### PR TITLE
TG-251 Test fixtures organisation 

### PR DESCRIPTION
**Pull request by denis256**
## Description

Included changes:
* moved fixtures to separated directory
* updated tests to reference moved files
* updated AWS tests to print errors in terminal to make debugging more easy

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [x] Update the docs.
- [x] Run the relevant tests successfully, including pre-commit checks.
- [x] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [x] Include release notes. If this PR is backward incompatible, include a migration guide.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

**Review by yhakbar — COMMENTED** [review]
Does every fixture need to be named `fixture-` if it's already in a `fixtures directory?

**Review by yhakbar — DISMISSED** [review]

File: test/fixtures/aws-provider-patch/main.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ provider "aws" {
 }
 
 module "example_module" {
-  source = "github.com/gruntwork-io/terragrunt.git//test/fixture-aws-provider-patch/example-module?ref=__BRANCH_NAME__"
+  source = "github.com/gruntwork-io/terragrunt.git//test/fixtures/fixture-aws-provider-patch/example-module?ref=__BRANCH_NAME__"
```
Comment (levkohimins) [inline test/fixtures/aws-provider-patch/main.tf:7]:
```suggestion
  source = "github.com/gruntwork-io/terragrunt.git//test/fixtures/aws-provider-patch/example-module?ref=__BRANCH_NAME__"
```

Please check if it should be renamed too.

Comment (denis256) [reply test/fixtures/aws-provider-patch/main.tf]: Yes, fixed this one too

File: cli/commands/scaffold/action.go [hunk 2]
```diff
@@ -306,7 +306,7 @@ func parseModuleUrl(ctx context.Context, opts *options.TerragruntOptions, vars m
 }
 
 // rewriteModuleUrl rewrites module url to git ssh if required
-// github.com/gruntwork-io/terragrunt.git//test/fixture-inputs => git::https://github.com/gruntwork-io/terragrunt.git//test/fixture-inputs
+// github.com/gruntwork-io/terragrunt.git//test/inputs => git::https://github.com/gruntwork-io/terragrunt.git//test/inputs
```
Comment (yhakbar) [inline cli/commands/scaffold/action.go:309]: These are wrong, but it doesn't really matter.

File: config/config.go [hunk 3]
```diff
@@ -572,11 +572,11 @@ func GetTerraformSourceUrl(terragruntOptions *options.TerragruntOptions, terragr
 //
 // and the terraform source is:
 //
-//	git::ssh://git@github.com/gruntwork-io/i-dont-exist.git//fixture-source-map/modules/app?ref=master
+//	git::ssh://git@github.com/gruntwork-io/i-dont-exist.git//source-map/modules/app?ref=master
```
Comment (yhakbar) [inline config/config.go:575]: Wrong, but doesn't matter.

**Review by yhakbar — APPROVED** [review]

**Review by levkohimins — APPROVED** [review]
LGTM!

**Pull request merged**